dash_app.py

```diff
@@ -116,28 +121,22 @@
     dbc.Tab(tab2_content, label="Graph", tab_id='graph-tab'),
     dbc.Tab(tab3_content, label='About')
 ],
-                active_tab='search-tab')
+    active_tab='search-tab')
 
 app.layout = dbc.Container(tabs, style=STYLE)
 
 
 def wrap_emoji(record, font_size):
-    return html.Div(
-        [
-            html.Div(record['emoji'],
-                     id=record['text'],
-                     style={'font-size': f'{font_size}em'},
-                     className='emoji'),
-            dcc.Clipboard(
-                target_id=record['text'],
-                style={
-                    'margin-left': '.75em',
-                    #     'position': 'relative',
-                    #       'margin': 'auto'
-                },
-                className='emoji'),
-            dbc.Tooltip(record['label'], target=record['text'])
-        ], )
+    return html.Div(children=[
+        html.P(record['emoji'],
+               id=record['text'],
+               style={'font-size': f'{font_size}em', 'margin-bottom': '0px'}),
+        dcc.Clipboard(target_id=record['text'],
+                      ),
+    ],
+
+        className='emoji',
```

Comment:
I think you may have working off my older code before I added  the `inline-block` code to the `typography.css` file and replaced `html.P` with just another div. Or did you think `html.P` is better and brought it back with intention?